.github/workflows/volta70.yml

```diff
@@ -0,0 +1,161 @@
+name: Reusable VOLTA70 workflow
+
+on:
+  workflow_call
+
+jobs:
+  PR_VOLTA70_CUDA1122_CUDA_LEFT_RIGHT_REL:
+    name: PR_VOLTA70_CUDA1122_CUDA_LEFT_RIGHT_REL
+    runs-on: [kk-env-cuda-11.2.2-gcc-8.4.0-latest]
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@v3
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@v3
+        with:
+          repository: kokkos/kokkos
+          ref: 4.4.00
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build,install}
+          cd kokkos/build
+          cmake -DCMAKE_CXX_COMPILER=$PWD/../bin/nvcc_wrapper \
+            -DCMAKE_CXX_FLAGS= \
+            -DCMAKE_EXE_LINKER_FLAGS= \
+            -DCMAKE_INSTALL_PREFIX=$PWD/../install \
+            -DKokkos_ENABLE_CUDA=ON \
+            -DKokkos_ARCH_VOLTA70=ON \
+            -DKokkos_ENABLE_TESTS=OFF \
+            -DKokkos_ENABLE_EXAMPLES=OFF \
+            -DCMAKE_VERBOSE_MAKEFILE=ON \
+            -DCMAKE_CXX_EXTENSIONS=OFF \
+            -DBUILD_SHARED_LIBS=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            -DKokkos_ENABLE_DEPRECATED_CODE_4=OFF \
+            -DKokkos_ENABLE_DEPRECATION_WARNINGS=OFF \
+            ..
+
+      - name: build_and_install_kokkos
+        working-directory: kokkos/build
+        run: make -j12 install
+
+      - name: configure_kokkos_kernels
+        run: |
+          mkdir -p kokkos-kernels/{build,install}
+          cd kokkos-kernels/build
+          cmake \
+            -DCMAKE_CXX_COMPILER=$PWD/../../kokkos/bin/nvcc_wrapper \
+            -DKokkos_DIR=$PWD/../../kokkos/install/lib64/cmake/Kokkos \
```

Comment:
We should shift to using `Kokkos_ROOT` , part of a follow up since this is already tested